wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Notify/ListManagerUserProfile.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace CDS\Modules\Notify;
+
+use WP_User;
+
+class ListManagerUserProfile
+{
+
+    public function __construct()
+    {
+    }
+
+    public static function register()
+    {
+        $instance = new self();
+
+        add_action('show_user_profile', [$instance, 'displayListManagerMeta']);
+        add_action('edit_user_profile', [$instance, 'displayListManagerMeta']);
```

Comment:
We can also add this permissions to GC Editors. That's fine, right? They still aren't able to see Notify settings, but they do get the List Manager page.

Reply:
Yes - we'll allow a super admin to add a GC Editor to send.

Reply:
So yes the Admin would setup the Notify settings as need be.